model/src/main/java/com/epam/repair/model/LastNameDTO.java

```diff
@@ -10,7 +10,7 @@
  */
 @Entity
 @Table(name = "last_name")
-public class LastName {
+public class LastNameDTO {
```

Comment:
Here can't be `dto` object because it's DB view

Reply:
Comment accepted.